modules/nw-ovn-kubernetes-migration.adoc

```diff
@@ -188,12 +188,44 @@ $ oc get machineconfig <config_name> -o yaml | grep ExecStart
 +
 where `<config_name>` is the name of the machine config from the `machineconfiguration.openshift.io/currentConfig` field.
 +
-The machine config must include the following update to the systemd configuration:
+The machine config can include the following update to the systemd configuration:
 +
 [source,plain]
 ----
 ExecStart=/usr/local/bin/configure-ovs.sh OVNKubernetes
 ----
++
+[NOTE]
+====
+The `configure-ovs.sh` Bash script has limited advanced configuration options, such as implementing post-installation changes to the OpenVSwitch/OVN-Kubernetes `br-ex` bridge network.
+====
+
+.. Optional: As a pre or post-installation task for installing a cluster on a bare-metal platform, you can use an alternative to `configure-ovs.sh` by creating a `MachineConfig` object that includes a customized `br-ex` bridge network configuration. After you reboot your cluster, the Kubernetes NMState Operator injects Ignition configuration files into each node in your cluster, so that each node received the `br-ex` bridge network configuration
```

Comment:
I see in [link](https://github.com/openshift/enhancements/pull/1440/files) that Day-2 activities do not require a reboot. I just want to check if this is correct and any other differences when compared to Day-0 activities.

Reply:
I also see the `NodeNetworkConfigurationPolicy` mentions in this link but I'm not following if it's required or just part of the discussion for options.